### ISSUE TITLE
make install target installs to wrong prefix on Gentoo

### ISSUE THREAD

**Issue by shawnanastasio** [nsz32/docklike-plugin#50]
When following the installation instructions in the README, the `make install` step appears to install the plugin to 
`/usr/lib/xfce4/panel/plugins/libdocklike.so`
but on my ppc64le Gentoo system, xfce4-panel expects the plugins to be in 
`/usr/lib64/xfce4/panel/plugins/`
instead. Manually copying the .so results in the plugin being found.

**Comment by nsz32** [comment]
Yes, installation path is hard coded in the compilation configuration, i will search for a correct variable.
Previous compilation system, using a variable as path, wasn't able to satisfy all distrib.
